app/bundles/QueueBundle/MauticQueueBundle.php

```diff
@@ -11,11 +11,82 @@
 
 namespace Mautic\QueueBundle;
 
+use Leezy\PheanstalkBundle\DependencyInjection\LeezyPheanstalkExtension;
+use Mautic\QueueBundle\Queue\QueueProtocol;
+use OldSound\RabbitMqBundle\DependencyInjection\Compiler\RegisterPartsPass;
+use OldSound\RabbitMqBundle\DependencyInjection\OldSoundRabbitMqExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
-/**
- * Class MauticQueueBundle.
- */
 class MauticQueueBundle extends Bundle
 {
+    /**
+     * @var array
+     */
+    private $localParams;
+
+    public function __construct(array $localParams)
+    {
+        $this->localParams = $localParams;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function build(ContainerBuilder $container)
+    {
+        parent::build($container);
+
+        $queueProtocol = $this->getQueueProtocol();
+
+        if (QueueProtocol::RABBITMQ === $this->getQueueProtocol()) {
```

Comment:
Not a big deal, but can we use $queueProtocol here?

Reply:
:-D